sample/src/main/java/io/fotoapparat/sample/ActivityJava.java

```diff
@@ -97,30 +106,23 @@ protected void onCreate(Bundle savedInstanceState) {
         fotoapparat = createFotoapparat();
 
         takePictureOnClick();
-        focusOnLongClick();
         switchCameraOnClick();
         toggleTorchOnSwitch();
         zoomSeekBar();
     }
 
     private Fotoapparat createFotoapparat() {
-        return Fotoapparat
-                .with(this)
-                .into(cameraView)
-                .previewScaleType(ScaleType.CenterCrop)
-                .lensPosition(back())
-                .frameProcessor(new SampleFrameProcessor())
-                .logger(loggers(
-                        logcat(),
-                        fileLogger(this)
-                ))
-                .cameraErrorCallback(new CameraErrorListener() {
-                    @Override
-                    public void onError(@NotNull CameraException e) {
-                        Toast.makeText(ActivityJava.this, e.toString(), Toast.LENGTH_LONG).show();
-                    }
-                })
-                .build();
+        return new Fotoapparat(this, cameraView, focusView, back(), ScaleType
```

Comment:
We introduced `builder` pattern in v1 and we kept it in v2 to help java users. We still believe builder it's better pattern for Java because of possibility to set only what you need and have the rest default. So `ActivityJava.java` is pretty good file to demonstrate that this can be used instead of the constructor

Reply:
Yeah, I though you guys were heading in the constructor direction. The Kotlin sample uses that with `FocusView`. 

Added the builder function as suggested.